XmlParser.lua

```diff
@@ -422,13 +426,14 @@ function XmlParser:parse(xml, parseAttributes)
     --and other auxiliar variables
     local f = {
         --string.find return
-        match = 0, 
+        match = 0,
         endMatch = 0,
-        text, end1, tagstr, end2,
+        -- text, end1, tagstr, end2,
```

Comment:
The goal to declare the fields without initializing them was to show which fields are expected at such a table.
I was also thinking about include documentation for such fields (but I don't have time now).
This way, I think that leaving the fields there would make it easier to have an overview of what such a table stores.
What do you think?

Reply:
It is better to place these variables to documentation because it is a bad style in Lua to put nils in table. Especially for those who use strict-checking as I do. I don't have any preferences about it. I can fix it to any code you like.

Reply:
Ok, so just leave them commented as you did.

Reply:
Ok, I will force-push with small fixes in a minute.